pkgs/cli/src/main.zig

```diff
@@ -35,6 +36,12 @@ const ZeamArgs = struct {
         beam: struct {
             help: bool = false,
             mockNetwork: bool = false,
+            metricsPort: u16 = 9667,
+        },
+        generate_prometheus_config: struct {
+            help: bool = false,
+            metricsPort: u16 = 9667,
+            output: []const u8 = "pkgs/metrics/prometheus/prometheus.yml",
```

Comment:
default behavior should be to dump in the current directory. This only works because you are running it inside the zeam repo, but people will get zeam via docker images, binary distributions, etc... so don't assume `pkgs/metrics/prometheus` will be there.